docs/proposal/adsmod.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Kmesh ads mode proposal
+authors:
+- "@LiZhenCheng9527"
+reviewers:
+approvers:
+
+creation-date: 2024-04-28
+
+---
+
+## Backgroud
+
+Because Kmesh provides a new data plane architecture, the control plane still uses istiod. So still need `ads`` for Kmesh and Istiod synchronisation.
+
+## Kmesh ads mode Function Design
+
+The process of synchronising Kmesh with istiod is the same as synchronising envoy with istiod:
+
+- **Create grpc links with istiod.**
+  After creating the grpc link, use the `streamAggregatedResources`` method to establish a link with istiod.
+- **Getting the initial configuration of the cluster.**
+  After establishing a link, Kmesh sends a clusterType empty ads message to istiod to get a snapshot of the current cluster configuration resources.
+- **Processing DiscoveryResponse response messages received from istiod.**
+  This is followed by the processing of the istiod return message. Kmesh processes the response differently depending on the typeUrl in the response.
+  CDS updates (if any) must always be pushed first.
```

Comment:
Add unordered list `-` to all below sentences

And add this target from envoy `In general, to avoid traffic drop, sequencing of updates should follow a make before break model, wherein:`